server/sonar-web/src/main/js/app/components/nav/global/GlobalNavMenu.tsx

```diff
@@ -75,7 +75,7 @@ export default class GlobalNavMenu extends React.PureComponent<Props> {
         <li>
           <Link
             className={classNames({ active })}
-            to={{ pathname: '/issues', query: { resolved: 'false' } }}>
+            to={{ pathname: '/issues', query: { resolved: 'false', myIssues: 'true' } }}>
```

Comment:
To Show only the issues assigned to currently logged in user for Codescan cloud.